R/cor_phylo.R

```diff
@@ -0,0 +1,450 @@
+
+# ================================================================================
+# ================================================================================
+
+# Main function
+
+# ================================================================================
+# ================================================================================
+
+
+
+
+#' Correlations among Multiple Traits with Phylogenetic Signal
+#' 
+#' This function calculates Pearson correlation coefficients for multiple continuous
+#' traits that may have phylogenetic signal, allowing users to specify measurement
+#' error as the standard error of trait values at the tips of the phylogenetic tree.
+#' Phylogenetic signal for each trait is estimated from the data assuming that trait
+#' evolution is given by a Ornstein-Uhlenbeck process.  Thus, the function allows the
+#' estimation of phylogenetic signal in multiple traits while incorporating
+#' correlations among traits. It is also possible to include independent variables
+#' (covariates) for each trait to remove possible confounding effects.
+#' \code{corphylo()} returns the correlation matrix for trait values, estimates
```

Comment:
cor_phylo?

Reply:
The documentation still needs a lot of work because of the name change and the interface change. I figured you'd want to know that nlopt was working though.